core/src/main/java/com/linecorp/armeria/client/AbstractRuleWithContentBuilder.java

```diff
@@ -36,7 +36,9 @@
  * @param <T> the response type
  */
 @UnstableApi
-public abstract class AbstractRuleWithContentBuilder<T extends Response> extends AbstractRuleBuilder {
+public abstract class AbstractRuleWithContentBuilder
+        <T extends Response, SELF extends AbstractRuleWithContentBuilder<T, SELF>>
```

Comment:
Global comment: Maybe `SELF` should come first?

Reply:
Thanks! Updated. 👍